s/sentry-python/latest/build_info.json

```diff
@@ -0,0 +1,10 @@
+{
+    "package_name":"sentry-python",
+    "github_url":"https://github.com/getsentry/sentry-python",
+    "package_dir":"s/sentry-python/latest/",
+    "version":"1.12.1",
+    "docker_build":false,
+     "1.12.1":{
```

Comment:
Have you checked the same build-script for other versions ?
If it is working for other versions, can you use some kind of regex here ?
Because, build will definitely fail if other versions selected.